From f301c8a1f97326abeae3a879b780b384e3f6941d Mon Sep 17 00:00:00 2001
From: Christian Berendt <berendt@osism.tech>
Date: Mon, 31 Jul 2023 09:48:23 +0200
Subject: [PATCH] ceilometer: use template module in "Copying over custom pipeline.yaml file"

Also rename task to "Copying over custom pipeline.yaml file" for
clarity.

Change-Id: I04e3eb9620830a15781f9bab2549b557a9d1d9cb
---

diff --git a/ansible/roles/ceilometer/tasks/config.yml b/ansible/roles/ceilometer/tasks/config.yml
index a4b46e9..26f4892 100644
--- a/ansible/roles/ceilometer/tasks/config.yml
+++ b/ansible/roles/ceilometer/tasks/config.yml
@@ -296,13 +296,13 @@
   delegate_to: localhost
   register: ceilometer_pipeline_file
 
-- name: Copying over pipeline.yaml
+- name: Copying over custom pipeline.yaml file
   vars:
     services_require_pipeline:
       - ceilometer-compute
       - ceilometer-central
       - ceilometer-notification
-  copy:
+  template:
     src: "{{ node_custom_config }}/ceilometer/pipeline.yaml"
     dest: "{{ node_config_directory }}/{{ item.key }}/pipeline.yaml"
     force: True
